source/macos/AdaptiveCards/AdaptiveCards/Views/ACRView.swift

```diff
@@ -18,6 +18,7 @@ class ACRView: ACRColumnView {
     private (set) var visibilityContext: ACOVisibilityContext?
     private (set) var accessibilityContext: ACSAccessibilityFocusManager?
     private (set) var cardWidth: CGFloat?
+    let queryManager = QueryManager()
```

Comment:
Here, every `ACRView` creates its own manager. Is that what you are looking for?

> Note: If you are looking for a single reference through the card, make it pass by ref.

Reply:
Did not put much thought into this because it's temporary just to test. This will not be the final place itll be used